packages/firebase_messaging/example/pubspec.yaml

```diff
@@ -2,6 +2,7 @@ name: firebase_messaging_example
 description: Demonstrates how to use the firebase_messaging plugin.
 
 dependencies:
+  firebase_analytics: any
```

Comment:
@collinjackson Why was this added? Is it recommended to also depend on `firebase_analytics` when you use `firebase_messaging`?